server/plugin/api_test.go

```diff
@@ -88,6 +90,18 @@ func TestPlugin_handleUserDisconnect(t *testing.T) {
 			GetDisconnectUserPostErr: errors.New("mockErr"),
 			DisconnectUserErr:        nil,
 		},
+		"Error decoding request body": {
+			httpTest: httpTestJSON,
+			request: testutils.Request{
+				Method: http.MethodPost,
+				URL:    "/api/v1/user/disconnect",
+				Body:   nil,
+			},
+			expectedResponse: testutils.ExpectedResponse{
+				StatusCode: http.StatusOK,
```

Comment:
why `StatusOK` here?

Reply:
We are logging the error in the logs not returning it. So, the status code is always OK